web/lib/embed.js

```diff
@@ -36,7 +36,8 @@ function loadTheGamma() {
           "libraries": g.providers.library("node_modules/thegamma-script/dist/libraries.json"),
           "shared": g.providers.rest("https://gallery-csv-service.azurewebsites.net/providers/listing", null, true),
           "olympics": g.providers.pivot(services + "pdata/olympics"),
-          "expenditure": g.providers.rest("https://govuk-expenditure.azurewebsites.net/expenditure") });
+          "expenditure": g.providers.rest("http://127.0.0.1:10039/expenditure") });
```

Comment:
Can you change this to the live service? I will deploy that before merging this PR.

Alternatively, if we wanted to do something more clever, we could test if the web is being served via localhost and use localhost in that case and the real service otherwise. This could save us a bit of pain with switching these all the time. I did something similar here: https://github.com/the-gamma/turing-2017/commit/d544d38aac7764e786389a53fe4ce768cefc3f66#diff-d9f173c99a1b5681cc4f624fc31ca706R4